### PR TITLE
Fix amino acid insertions

### PR DESCRIPTION
**Pull request by JonasKellerer**
Resolves the issue, that for the large dataset, amino acid insertions are not read in properly. It seems to be an issue with duckdb.

### CODE REVIEW

**Comment by JonasKellerer** [comment]
I now moved the fix to the insertion column. 
With the large dataset it takes roughly the same time.

File: src/silo/common/string_utils.cpp [hunk 1]
```diff
@@ -19,4 +19,12 @@ std::vector<std::string> splitBy(const std::string& value, const std::string_vie
    return splits;
 }
 
+std::string removeSymbol(const std::string& value, const std::string_view symbol) {
+   std::string result = value;
+   const auto symbol_position = result.find(symbol);
+   if (symbol_position != std::string::npos) {
+      result.erase(symbol_position, symbol.size());
+   }
+   return result;
```
Comment (Taepper) [inline src/silo/common/string_utils.cpp:28]: This only removes up to one quote. Lets also cover the case where on string is completely quoted

```
return value.erase(
      std::remove(value.begin(), value.end(), '\"'),
      value.end()
   );
```

Comment (Taepper) [reply src/silo/common/string_utils.cpp:28]: (remove all " from the string, not only the first one found)

**Review by Taepper — APPROVED** [review]

File: src/silo/common/string_utils.cpp (hunk 1 above)
Comment (JonasKellerer) [reply src/silo/common/string_utils.cpp:26]: I changed it, so it removes all. Additionally, I added tests to verify the behavior.

**Pull request merged**